diff --git a/test_custom_types.py.before b/test_custom_types.py
index 41b79a9..3172ed2 100644
--- a/test_custom_types.py.before
+++ b/test_custom_types.py
@@ -255,7 +255,7 @@ class TestBSONFallbackEncoder(unittest.TestCase):
 
 class TestBSONTypeEnDeCodecs(unittest.TestCase):
     def test_instantiation(self):
-        msg = "Can't instantiate abstract class .* with abstract methods .*"
+        msg = "Can't instantiate abstract class"
         def run_test(base, attrs, fail):
             codec = type('testcodec', (base,), attrs)
             if fail:
